tests/api_test.py

```diff
@@ -28,10 +28,10 @@ def test_extracting_data_from_url_type_2(self):
 
     def test_extracting_data_from_url_type_3(self):
         """The third type of URL is in the format: "https://www.youtube.com/c/@{username}"""
-        api = YoutubeApi("https://www.youtube.com/c/@EnglishClass101")
```

Comment:
BTW, why it was unstable?

Reply:
I don't mind. Now in the GitHub Actions container, the YouTube API returns a different channel id for that channel. Maybe it somehow depends on the location of the server.

Reply:
I am surprised that one YouTube channel can have several IDs.